src/core/server/integration_tests/saved_objects/migrations/multiple_kibana_nodes.test.ts

```diff
@@ -99,6 +99,9 @@ async function createRoot({ logFileName }: CreateRootConfig) {
   return root;
 }
 
+// suite is very long, the 10mins default can cause timeouts
+jest.setTimeout(15 * 60 * 1000);
```

Comment:
The test suite is taking 5mins on average, according to the CI stats. Meaning that the default 10mins timeout we're using for our integration test may not be sufficient in some scenarios, which could also explain the few timeouts we saw.